modules/example/src/smithy4s/example/StructureWithRefinedTypes.scala

```diff
@@ -9,8 +9,8 @@ object StructureWithRefinedTypes extends smithy4s.ShapeTag.Companion[StructureWi
   val hints : smithy4s.Hints = smithy4s.Hints.empty
 
   implicit val schema: smithy4s.Schema[StructureWithRefinedTypes] = struct(
-    Age.schema.optional[StructureWithRefinedTypes]("age", _.age),
-    PersonAge.schema.optional[StructureWithRefinedTypes]("personAge", _.personAge),
+    Age.schema.optional[StructureWithRefinedTypes]("age", _.age).addHints(smithy.api.Default()),
+    PersonAge.schema.optional[StructureWithRefinedTypes]("personAge", _.personAge).addHints(smithy.api.Default()),
```

Comment:
Going to see why these showed up... they shouldn't have

Reply:
The reason these showed up is that smithy 2.0 does the following when loading v1 models:

```
Inject an appropriate `@default` trait on structure members that targeted
      shapes with a default zero value and were not marked with the `@box`
      trait
```

Not sure how we want to proceed here.. especially since the default seems to be empty? I guess that indicates a null default? In that case, this could be problematic with required fields since they'll get the empty default trait as well.. which would indicate that a `null` default is valid, where it isn't.